src/main/server.ts

```diff
@@ -1,17 +1,27 @@
 #!/usr/bin/env node
-const { Logger } = require('@hmcts/nodejs-logging')
+const appInsights = require('applicationinsights');
+const config = require('@hmcts/properties-volume').addTo(require('config'));
+const { Logger } = require('@hmcts/nodejs-logging');
```

Comment:
you can do same if they have type defs if not this is fine.

Reply:
done